addon/components/-private/base-pop-menu.js

```diff
@@ -93,7 +102,7 @@ export default class BasePopMenuComponent extends Component {
   /**
    * Modifiers to the popper to set it's default behavior
    */
-  _popperModifiers = {
+  _defaultPopperModifiers = {
```

Comment:
We should make this a module variable instead of a class instance field, so we only create the object once